src/components/skills-quiz/tests/SkillsQuizPage.test.jsx

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
```

Comment:
Extracted majority (95% of logic) from the `SkillsQuiz` test file, just reuses test to render the parent `SkillsQuizPage` component correctly